test/FixedGovLst.t.sol

```diff
@@ -177,6 +177,33 @@ contract Constructor is FixedGovLstTest {
     assertEq(fixedLst.SHARE_SCALE_FACTOR(), lst.SHARE_SCALE_FACTOR());
     assertEq(fixedLst.name(), tokenName);
     assertEq(fixedLst.symbol(), tokenSymbol);
+    assertEq(fixedLst.decimals(), 18);
+    assertEq(fixedLst.version(), "2");
+  }
+}
+
+contract DefaultDelegatee is FixedGovLstTest {
+  function test_DefaultDelegatee() public view {
+    assertEq(fixedLst.defaultDelegatee(), lst.defaultDelegatee());
+  }
+}
+
+contract DepositForDelegatee is FixedGovLstTest {
+  function testFuzz_DepositForDelegatee(address _delegatee) public view {
+    assertEq(fixedLst.depositForDelegatee(_delegatee), lst.depositForDelegatee(_delegatee));
+  }
+}
+
+contract FetchOrInitializeDepositForDelegatee is FixedGovLstTest {
+  function testFuzz_FetchOrInitializeDepositForDelegatee(address _delegatee) public {
```

Comment:
Needs a more expressive name for scopelint

Reply:
Done: 3d7a520307fe8a12df63ece44b2671ceaec41bb5